widgets/price-table/tpl/atom.php

```diff
@@ -24,15 +25,15 @@
 				<div class="ow-pt-per"><?php echo esc_html($column['per']) ?></div>
 			</div>
 
-			<?php if(!empty($column['image'])) : ?>
-				<div class="ow-pt-image">
-					<?php $this->column_image($column) ?>
-				</div>
-			<?php endif; ?>
+            <div class="ow-pt-image">
```

Comment:
Main reason is that if the image row is present in one feature and we're doing the row height equalization, there needs to be an element on which to set the height that matches the height of the image. (I hope that made sense :sweat_smile: ) So perhaps I'll change this to output if at least one column has an image.